apps/shipments/serializers.py

```diff
@@ -414,11 +417,14 @@ def create(self, validated_data):
 class ChangesDiffSerializer:
     relation_fields = settings.RELATED_FIELDS_WITH_HISTORY_MAP.keys()
     excluded_fields = ('history_user', 'version', 'customer_fields', 'geometry',
-                       'background_data_hash_interval', 'manual_update_hash_interval')
+                       'background_data_hash_interval', 'manual_update_hash_interval', 'shipment', )
 
     # Enum field serializers
     stateEnumSerializer = UpperEnumField(TransitState, lenient=True, ints_as_names=True, read_only=True)
     exceptionEnumSerializer = UpperEnumField(ExceptionType, lenient=True, ints_as_names=True, read_only=True)
+    file_typeEnumSerializer = UpperEnumField(FileType, lenient=True, ints_as_names=True, read_only=True)
```

Comment:
Please be consistent when casing variables - https://www.python.org/dev/peps/pep-0008/#function-and-variable-names